spec/ruby/library/matrix/reflexive_spec.rb

```diff
@@ -0,0 +1,29 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'matrix'
+
+describe "Matrix.reflexive?" do
+  it "returns true for a reflexive Matrix" do
+    Matrix[[1, 2, 3], [4, 1, 3], [5, 3, 1]].reflexive?.should be_true
+  end
+
+  it "returns true for a 0x0 empty matrix" do
+    Matrix.empty.symmetric?.should be_true
```

Comment:
wrong method

Reply:
Thank you for being so patient. Most of the code were follow the style of method "symmetric?". I've learned 1-0 matrix, and it seems to me that reflexive relation cannot applied in a general matrix, this method might not be very helpful, but I'll still fix this problem.